### PR TITLE
Link contributer guidelines to readme | borrowed issue and PR templates 

### PR DESCRIPTION
**Pull request by a2un**
Link contributer guidelines to readme

### CODE REVIEW

**Comment by ananyo2012** [comment]
@a2un Why are there so many files ? We only needed 2.

**Comment by a2un** [comment]
These are different templates (2 of each kind)

**Comment by a2un** [comment]
Okay I shall keep the rest in my local and update just with the conversational one.

**Comment by a2un** [comment]
@ananyo2012  @realslimshanky  updated the pull request.

File: .github/conversational/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+**Note: for support questions, please use stackoverflow**. This repository's issues are reserved for feature requests and bug reports.
```
Comment (vipulgupta2048) [inline .github/conversational/ISSUE_TEMPLATE.md]: The communication channels are Discord and mailing list. 
Please edit thoroughly before you procure information from other sources. Just like the proverb in Hindi. :laughing:

File: .github/conversational/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* **Do you want to request a *feature* or report a *bug*?**
+
+
+
+* **What is the current behavior?**
+
+
+
+* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via
+https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/tpl:AvJOMERrnz94ekVua0u5).
```
Comment (vipulgupta2048) [inline .github/conversational/ISSUE_TEMPLATE.md]: I don't how plunker helps here, other than making things more complicated than they already are. I feel implementing this for a simple static blog generated by Pelican is bit of an overkill. 
Simple - Give an appropriate description of the bug, steps to reproduce it and attach screenshots or use similar means for the same.

File: .github/conversational/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+* **Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines
+- [ ] Tests for the changes have been added (for bug fixes / features)
```
Comment (vipulgupta2048) [inline .github/conversational/PULL_REQUEST_TEMPLATE.md]: What kind of tests do you mean here?

File: README.md [hunk 4]
```diff
@@ -14,6 +14,10 @@ $ ./develop_server.sh start
 
 The contents should be visible at [http://localhost:8000/blog/](http://localhost:8000/blog/)
 
+## Contributing to the Blog site
+
+Please read our [Contribution Guidelines](CONTRIBUTING.md) to Contribute to the Blog site
```
Comment (vipulgupta2048) [inline README.md]: Capital letter in the middle of sentences doesn't look good anywhere and neither does grammatical errors in README.md. 

Rephrasing 

**Want to contribute?**

Thank you for your you interest in contributing to PyCon India blog site, please do read our [Contribution Guidelines](CONTRIBUTING.md) to get started. Submit an issue in the Issues section for reporting/fixing bugs and to request new features.

File: .github/conversational/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+* **What is the motivation / use case for changing the behavior?**
+
+
+
+* **Please tell us about your environment:**
+  
+  - Version: 2.0.0-beta.X
+  - Browser: [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
+  - Language: [all | TypeScript X.X | ES6/7 | ES5 | Dart]
```
Comment (vipulgupta2048) [inline .github/conversational/ISSUE_TEMPLATE.md]: Markdown / reStructered text / Python

File: .github/conversational/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+
+
+* **What is the motivation / use case for changing the behavior?**
+
+
+
+* **Please tell us about your environment:**
+  
+  - Version: 2.0.0-beta.X
```
Comment (vipulgupta2048) [inline .github/conversational/ISSUE_TEMPLATE.md]: We don't have releases.

**Comment by vipulgupta2048** [comment]
Reviewed till https://github.com/pythonindia/inpycon-blog/pull/135/commits/0917c29740f0222ed382c9d6914b938664fed01e

**Few points**
1. Do tick the edit for maintainers checkbox, it would be much efficient if we correct the trivial details ourselves that doesn't require reviews.
2. Look before you copy, not very source is completely right. Always modify content as per your use case.
3. Great work.

**Comment by a2un** [comment]
Closing pull requests lot of errors noted..Will make fresh changes and raise another

**Pull request closed without merging**